src/main/java/io/gravitee/repository/management/api/AlertTriggerRepository.java

```diff
@@ -30,4 +30,6 @@
     Set<AlertTrigger> findAll() throws TechnicalException;
 
     List<AlertTrigger> findByReference(String referenceType, String referenceId) throws TechnicalException;
+
+    List<AlertTrigger> findByReferenceAndReferenceIds(String referenceType, List<String> referenceIds) throws TechnicalException;
```

Comment:
Should we rename it to `findByReferenceAndReferenceIds` for consistancy with `findByReference`?

Also, `findByReference` could have a default implementation then, something like: 
```java
    default List<AlertTrigger> findByReference(String referenceType, String referenceId) throws TechnicalException {
        return findByReferenceAndReferenceIds(referenceType, List.of(referenceId));
    }
```